### DISCUSSION TITLE
Where is the docs repository?

### DISCUSSION THREAD

**Discussion by tiagof** [filamentphp/filament#3609] [Help] [answered]
Hi,

I'd like to update the docs (via PR, of course) to improve the "publishing the translations" section.
I believe it should include the filament-support translations as well.

Can anyone point me to the correct repository and path?
Thanks

**Comment by danharrin** [comment, marked as answer]
The docs are all in this repository. Each package in /packages has a /docs directory.

**Reply by tiagof** [reply]
(was looking under packages/support)
Found it. Thx!
